corehq/form_processor/backends/sql/processor.py

```diff
@@ -33,6 +33,7 @@ def store_attachments(cls, xform, attachments):
                 attachment_id=uuid.uuid4(),
                 content_type=attachment.content_type,
             )
+            xform_attachment.blob_bucket = xform_attachment.blobdb_bucket()
```

Comment:
On second thought, from a space perspective it does seem redundant to save this value in `blob_bucket` since it's simply `"form/" + self.attachment_id.hex`. It's more efficient (in terms of space) to not set this here since `blobdb_bucket()` now returns the correct value by default. Thoughts?